gradle/libs.versions.toml

```diff
@@ -15,7 +15,7 @@ okhttp = "4.12.0"
 okio = "3.15.0"
 paparazzi = "1.3.5"
 roborazzi = "1.45.1"
-skiko = "0.9.4.2"
+skiko = "0.9.21"
```

Comment:
Is updating Skiko required to support the new target? We keep our Skiko version the same as the latest stable Compose to prevent issues